### ISSUE TITLE
Pending MoneyMan transactions don't show up in "Transactions -> Omni Layer" list

### ISSUE THREAD

**Issue by msgilligan** [OmniLayer/omnicore#310]
I'm running Omni Core Qt on the TestNet. When I send BTC to the MoneyMan address. I see the pending transaction in "Transactions -> Bitcoin" list, but I don't see it in the "Transactions -> Omni Layer" list. It seems to me that something should be there showing the pending Omni activity.

**Comment by dexX7** [comment]
It's a duplicate of #265, although you mention unconfirmed transactions here, too.

**Comment by dexX7** [comment]
I've been thinking a bit about this and the biggest hurdle right now is that we don't have a proper representation of Exodus purchases. We can parse transactions, that's one thing, but it adds a special meaning, similar to BTC/MSC DEx trades.

**Comment by zathras-crypto** [comment]
I guess I've not really seen this as a high priority as it will never be an issue in the future outside of testing, and original Exodus purchases were so long ago I doubt many current Omni Core users participated back then.

Thus I'm not hugely inclined to add further complexity at this stage, but I don't feel strongly about it so if my view is not shared and we want to prioritize adding support for Exodus transactions in the UI then I'll go with the flow, but it's worth noting we don't even have RPC support for Exodus transactions which should arguably come first.

**Comment by dexX7** [comment]
I agree. I'd be much more inclined to tackle this in the context of a more generalized approach to handle multiple effects etc.
